portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,13 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Kenneth Kenfack's Portfolio</h1>
+      <p>This is my portfolio. It's all about me a Cameroonian king. Growing up to be amazing. I'm currently a sophmore at umbc</p>
```

Comment:
Aside: if I was reviewing this for my work project, I'd be annoying and ask you to use conventional grammar because we have a rule somewhere that says "Text and comments should be complete grammatically correct sentences."